### PR TITLE
add-vni-to-kvmtpr

### PR DESCRIPTION
**Pull request by calvix**
add-vni-to-kvmtpr #14

### CODE REVIEW

File: pkg/apis/cluster/v1alpha1/kvm_types.go [hunk 1]
```diff
@@ -208,6 +209,14 @@ type KVMSpecKVMNode struct {
 	Memory string  `json:"memory" yaml:"memory"`
 }
 
+type KVMSpecKVMNetwork struct {
+	Flannel KVMSpecKVMNetworkFlannel `json:"flannel" yaml:"flannel"`
+}
+
+type KVMSpecKVMNetworkFlannel struct {
+	VNI int `json:"vni" yaml:"vni"`
+}
```
Comment (kopiczko) [inline pkg/apis/cluster/v1alpha1/kvm_types.go]: Did you rerun `./scripts/gen.sh`? I think DeepCopy functions should be regenerated.

Comment (xh3b4sd) [reply pkg/apis/cluster/v1alpha1/kvm_types.go]: Good shout. Forgot about it.

Comment (calvix) [reply pkg/apis/cluster/v1alpha1/kvm_types.go]: I have not, that would be good info for Readme :)

Comment (xh3b4sd) [reply pkg/apis/cluster/v1alpha1/kvm_types.go]: Yeah 100%. There is a PR but it takes apparently a long time to merge it. #hinthint

Comment (calvix) [reply pkg/apis/cluster/v1alpha1/kvm_types.go]: https://github.com/giantswarm/apiextensions/pull/15

**Pull request merged**